### ISSUE TITLE
No Notepad plz

### ISSUE THREAD

**Issue by pwn0r** [henrypp/simplewall#455]
Can we use something better than notepad for viewing the log file? It always opens windows builtin notepad even though I have notepad++ and even linked *.log files to open there as well? I.e. if I double-click on that file in explorer I get notepad++, but from simplewall UI it still opens basic notepad.

**Comment by hesss5** [comment]
download CSVFileView
https://www.nirsoft.net/utils/csv_file_view.html
extract to same  same folder as simplewall.exe
open simplewall.ini in section of [simplewall] add following line
LogViewer=.\csvfileview.exe
restart simplewall

**Comment by henrypp** [comment]
https://github.com/henrypp/simplewall/wiki/FAQ#how-can-i-use-3-rd-party-software-to-view-log

**Comment by hesss5** [comment]
however if you still want use notepad++ try changing exe and path of bellow registry file
bellow registry set Notepad2 as default notepad so SW will open notepad2



Windows Registry Editor Version 5.00

[HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Image File Execution Options\notepad.exe]
"Debugger"="\"C:\\Program Files\\Notepad2\\Notepad2.exe\" /z"

**Comment by pwn0r** [comment]
thanks for everyone. In short I used my own way. Since this INI command is available, I just used it to point to notepad++.exe <3 Now it works.

Just in case -- one reason to use N++ instead of built-in -- it automatically detects that the file was appended and will ask a user if he needs to reload it. This way I dont need to reopen a log file every three minutes if I'm testing something. Only open once and reload as necessary.
